### ISSUE TITLE
Error in running notebook crop_cycles.ipynb

### ISSUE THREAD

**Issue by click2cloud-sagarB** [microsoft/farmvibes-ai#40]
Dear Farmvibes Team,
While running notebook _crop_cycles.ipynb_ the task  "chunk_onnx.chunk_raster"  got failed and raise value error   " **dim size cannot be smaller than chip size**"
I have attached the screenshot of task which was failed along with its log for better understanding. How can I resolve this issue ?

![1](https://user-images.githubusercontent.com/115791095/218683072-752816db-633d-42cb-962d-d4f4e8532f37.JPG)

![2](https://user-images.githubusercontent.com/115791095/218683116-1f8d81ca-a424-412d-a96c-2a1f5f57c353.JPG)

**Comment by rafaspadilha** [comment]
Hi @click2cloud-sagarB . Thank you for raising this issue. 
This happens when the geometry/region of interest is small. When running the onnx model for the crop cycle count, the workflow will generate chips of size 100x100 pixels, however it seems that the region of interest has one of its dimensions of 51 pixels. 

To unblock you on this, you can alter the `notebooks/crop_cycles/crop_cycles.yaml` to the following file:

```
name: crop_cycles
sources:
  user_input:
    - spaceeye.user_input
sinks:
  cycles: chunk_onnx.raster
parameters:
  pc_key:
  model_file: count_cycles.onnx
  step:  25
tasks:
  spaceeye:
    workflow: data_ingestion/spaceeye/spaceeye_interpolation
    parameters:
      pc_key: "@from(pc_key)"
  ndvi:
    workflow: data_processing/index/index
    parameters:
      index: ndvi
  chunk_onnx:
    workflow: data_processing/chunk_onnx/chunk_onnx
    parameters:
      model_file: "@from(model_file)"
      step: "@from(step)"
edges:
  - origin: spaceeye.raster
    destination:
      - ndvi.raster
  - origin: ndvi.index_raster
    destination:
      - chunk_onnx.rasters
```

This will define a smaller chip size (step of 25x25 pixels) and pass it as a parameter to `chunk_onnx` workflow. 

In the next release, we will improve the error message for clarity and modify the `crop_cycle.yaml` file to reflect this changes.
Once again, thank you for raising this issue. 

Let me know if this solves it for your case.

**Comment by rafaspadilha** [comment]
Closing this issue due to the fix in PR #44 .
Let us know if you still find any issues while running.

**Comment by click2cloud-sagarB** [comment]
Thanks! @rafaspadilha . After altering crop_cycles.yaml to this file you provided it works perfectly.
